codegen/allapigen.py

```diff
@@ -59,3 +61,5 @@
     allapigen.generate(version, static_infos)
     t2 = time()
     print(f"Time to generate APIs: {t2 - t1:.2f} seconds")
+    pyfluent.WRITE_API_SEARCH_OBJECTS_FILE = True
+    _search("", version="242")
```

Comment:
We can probably now remove the txt files in api_tree folder from the version control. They should be packaged just like other generated files.

Reply:
We probably can remove the flag `WRITE_API_SEARCH_OBJECTS_FILE` also - I don't see any use of it ATM.

Reply:
Yes, I agree.

Reply:
Done. Thank you.

Reply:
Done. Thank you.